python/mxnet/libinfo.py

```diff
@@ -76,6 +76,33 @@ def find_lib_path():
         os.environ['PATH'] = os.environ['PATH'] + ';' + os.path.dirname(lib_path[0])
     return lib_path
 
+def find_include_path():
+    """Find MXNet dynamic library files.
```

Comment:
Would be great to know how is it exactly different from the above function. Both mention the same thing "Find MXNet dynamic library files."

Reply:
Changed